source/world/level/path/BinaryHeap.cpp

```diff
@@ -71,7 +71,7 @@ void BinaryHeap::downHeap(int num)
 		float var9;
 		if (var5 >= m_count) {
 			var8 = nullptr;
-			var9 = INFINITY;
+			var9 = std::numeric_limits<float>::infinity();
```

Comment:
Why was this changed?

Reply:
I know, it's because `INFINITY` was added shockingly late, in C++11.

Reply:
In classic C++ fashion...